app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,49 @@
+class AnswersController < ApplicationController
+
+  def new
+    @question = Question.find(params[:id])
+    @answer = @question.answers.new
```

Comment:
You could just do `Answer.new` here - you don't need to attach - it's just for the form.